LibraryTests/Samples/ReversibleLogicSynthesisTests.qs

```diff
@@ -0,0 +1,82 @@
+// Author: Mathias Soeken, EPFL (Mail: mathias.soeken@epfl.ch)
```

Comment:
I apologize for the hassle, but I believe that we need for the copyright header to match the headers on other files; would you be willing to modify the PR accordingly? Thanks!

Reply:
Okay, will fix that.